pymchelper/readers/shieldhit/reader_bdo2019.py

```diff
@@ -47,41 +48,50 @@ def read_data(self, estimator):
                 if payload_len == 1:
                     payload = payload[0]
 
-                logger.debug("Read token {:s} (0x{:02x}) value {} type {:s} length {:d}".format(
-                    SHBDOTagID(token_id).name,
-                    token_id,
-                    raw_payload,
-                    token_type.decode('ASCII'),
-                    payload_len
-                ))
+                try:
+                    token_name = SHBDOTagID(token_id).name
+                    logger.debug("Read token {:s} (0x{:02x}) value {} type {:s} length {:d}".format(
+                        token_name,
+                        token_id,
+                        raw_payload,
+                        token_type.decode('ASCII'),
+                        payload_len
+                    ))
+                except ValueError:
+                    logger.info("Skipping token (0x{:02x}) value {} type {:s} length {:d}".format(
```

Comment:
maybe add why it was skipped to user?

Reply:
What do you think about:
```
"Found unknown token (0x{:02x}) value {} type {:s} length {:d}, skipping"
```
?

Reply:
See https://github.com/DataMedSci/pymchelper/pull/402/commits/a8a85a2d1e749906297c1f84abf2a54c1acb7342

Reply:
much better, yes. thanks